runtime/src/info.rs

```diff
@@ -17,6 +18,7 @@ impl FwInfoCmd {
         let runtime_svn = handoff.rt_svn()?;
```

Comment:
Would failure to report FwInfo be classified as fatal?

Reply:
I don't think it should be fatal. If the caller considers it fatal they can issue the FIPS shutdown command.

Reply:
I agree I don't think we'd want it to be fatal. Currently it would bubble that result back to the mailbox command handler and report a non-fatal error here:
![image](https://github.com/chipsalliance/caliptra-sw/assets/139178705/c634506c-feec-4310-9449-47462682caca)

Reply:
Will all integrators enable FIPS mode without exception ?

Reply:
We may have to find someone else to help answer that. I wasn't aware there was a mode for FIPS. What does this mode affect?

Reply:
Also if this fails they can call DisableAttestation, fail to boot, or fail to forward requests to Caliptra to mitigate any possible security issues.